src/platform/silabs/SiWx917/wifi/wfx_notify.cpp

```diff
@@ -21,6 +21,7 @@
 
 #include "AppConfig.h"
 
+#include "BaseApplication.h"
```

Comment:
I am a bit weary of bringing BaseApplication into `src/platform/silabs`

The app layer uses the platform layer but the platform code shouldn't be pulling app layer. This created some circular dependency.

I believe this is already the case in some other files in `platform/silabs/*/wifi/` files. We definitely will have to restructure that at some point.

Reply:
correct, checking the commissioning window status, since that should be fine